src/Router.tsx

```diff
@@ -1,25 +1,37 @@
 import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-import Login from './Login/login';
 import HomePage from './Home/HomePage';
+import VerificationLandingPage from './Login/VerificationLandingPage';
 
 interface RouterProps {
 
 }
 
+// const PrivateRoute = ({ component: Component, ...rest }) => (
```

Comment:
This is a potential function to access guarded pages by making an authentication check. It's commented out because we didn't have authentication logic when I wrote it. I'll merge it in, and remove it later if there's a better way to do auth guarding.